astropy/io/ascii/tests/test_c_reader.py

```diff
@@ -13,6 +13,7 @@
 import pytest
 from numpy import ma
 
+from numpy import testing as npt
```

Comment:
I'm not sure if I would use this `npt` alias, but it is used in some of our tests already, so there is precedent for this.

Reply:
–1 on adopting this. We have discussed this in one of the various PRs addressing the same issue before, and already concluded that `from numpy.testing import assert_allclose` etc. is the preferred style (and already used with only the aforementioned 1 or 2 exceptions).